keyboards/keeblounge/concord45/keymaps/default/keymap.c

```diff
@@ -0,0 +1,71 @@
+     * ├────┬─┴─┬─┴──┬┴───┴───┴───┴───┴───┴───│───┴┬───┘ │ ↑ │ └───┘
+     * │LCtl│OS │LAlt│             Space      │ Ln │ ┌───┤───├───┐
+     * └────┴───┴────┴────────────────────────┴────┘ │ ← │ ↓ │ → │
+     *                                               └───┴───┴───┘
+     */
+    [_BASE] = LAYOUT(KC_ESC, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_DEL, KC_BSPC, KC_PGUP, KC_TAB, KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_QUOTE, KC_ENTER, KC_PGDN, KC_LSFT, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_RSFT, KC_UP, MO(_FN), KC_LCTL, KC_LGUI, KC_LALT, KC_SPACE, MO(_LN), KC_LEFT, KC_DOWN, KC_RIGHT),
+    /* Main Numbers, Symbols & Function Layer (MOMENTARY)
+     * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┐   ┌───┐
+     * │   │ ! │ @ │ # │ $ │ % │ ^ │ & │ * │ [ │ ] │ \ │   │   │Hme│
+     * ├───┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴───┤   ├───┤
+     * │    │ 1 │ 2 │ 3 │ 4 │ 5 │ 6 │ 7 │ 8 │ 9 │ 0 │      │   │End│
+     * ├────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴──┬───┘   ├───┤
+     * │      │ ( │ ) │ ; │ . │   │ - │ + │ * │ / │ =  │ ┌───┐ │   │
+     * ├────┬─┴─┬─┴──┬┴───┴───┴───┴───┴───┴───│───┴┬───┘ │   │ └───┘
+     * │    │   │    │                        │    │ ┌───┤───├───┐
+     * └────┴───┴────┴────────────────────────┴────┘ │   │   │   │
+     *                                               └───┴───┴───┘
+     */
+    [_FN] = LAYOUT(XXXXXX, KC_EXLM, KC_AT, KC_HASH, KC_DOLLAR, KC_PERCENT, KC_CIRC, KC_AMPR, KC_ASTR, KC_LBRC, KC_RBRC, KC_BSLS, XXXXXX, KC_HOME, XXXXXX, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_ENTER, KC_END, XXXXXX, KC_LPRN, KC_RPRN, KC_SCLN, KC_DOT, XXXXXX, KC_MINS, KC_PLUS, KC_ASTR, KC_SLSH, KC_EQL, XXXXXX, KC_NO, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX),
+
+    /* ALTERNATE Function layer (MOMENTARY)
+     * ┌────┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┐
+     * │ PWR│F1 │F2 │F3 │F4 │F5 │F6 │   │   │   │   │PRV│NXT│   │VL+│
+     * ├────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴───┼───┼───┤
+     * │ SLP │F7 │F8 │F9 │F10│F11│F12│   │   │   │   │ PLAY │   │VL-│
+     * ├─────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴──┬───┴───│───│
+     * │ WAKE  │   │   │   │   │   │   │   │   │   │    │ ┌───┐ │   │
+     * ├────┬──┴─┬─┴──┬┴───┴───┴───┴───┴───┴───│───┴┬───┘ │   │ └───┘
+     * │RSET│    │    │                        │    │ ┌───│───│───┐
+     * └────┴────┴────┴────────────────────────┴────┘ │   │   │   │
+     *                                                └───┴───┴───┘
+     */
+    [_LN] = LAYOUT(KC_PWR, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, XXXXXX, XXXXXX, XXXXXX, XXXXXX, KC_MPRV, KC_MNXT, KC_VOLU, KC_SLEP, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, XXXXXX, XXXXXX, XXXXXX, XXXXXX, KC_MPLY, KC_VOLD, KC_WAKE, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, RESET, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX),
```

Comment:
Tidies up the block comments, and corrects the error. Your keymap is still using the `XXXXXX`, which is deleted, so the keymap doesn't compile.

```suggestion
    /* Default QWERTY layer
     * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┐   ┌───┐
     * │Esc│ Q │ W │ E │ R │ T │ Y │ U │ I │ O │ P │Del│BkS│   │PgU│
     * ├───┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴───┤   ├───┤
     * │Tab │ A │ S │ D │ F │ G │ H │ J │ K │ L │ ' │Enter │   │PgD│
     * ├────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴──┬───┘   ├───┤
     * │LShift│ Z │ X │ C │ V │ B │ N │ M │ , │ . │Sft/│ ┌───┐ │Fn │
     * ├────┬─┴─┬─┴──┬┴───┴───┴───┴───┴───┴───┼───┴┬───┘ │ ↑ │ └───┘
     * │LCtl│OS │LAlt│         Space          │ Ln │ ┌───┼───┼───┐
     * └────┴───┴────┴────────────────────────┴────┘ │ ← │ ↓ │ → │
     *                                               └───┴───┴───┘
     */
    [_BASE] = LAYOUT(
        KC_ESC,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_DEL,  KC_BSPC, KC_PGUP,
        KC_TAB,  KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_QUOT,          KC_ENT,  KC_PGDN,
        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_RSFT,          KC_UP,   MO(_FN),
        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                    MO(_LN),          KC_LEFT, KC_DOWN, KC_RGHT
    ),

    /* Main Numbers, Symbols & Function Layer (MOMENTARY)
     * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┐   ┌───┐
     * │   │ ! │ @ │ # │ $ │ % │ ^ │ & │ * │ [ │ ] │ \ │   │   │Hme│
     * ├───┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴───┤   ├───┤
     * │    │ 1 │ 2 │ 3 │ 4 │ 5 │ 6 │ 7 │ 8 │ 9 │ 0 │      │   │End│
     * ├────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴──┬───┘   ├───┤
     * │      │ ( │ ) │ ; │ . │   │ - │ + │ * │ / │ =  │ ┌───┐ │   │
     * ├────┬─┴─┬─┴──┬┴───┴───┴───┴───┴───┴───┼───┴┬───┘ │   │ └───┘
     * │    │   │    │                        │    │ ┌───┼───┼───┐
     * └────┴───┴────┴────────────────────────┴────┘ │   │   │   │
     *                                               └───┴───┴───┘
     */
    [_FN] = LAYOUT(
        XXXXXXX, KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC, KC_CIRC, KC_AMPR, KC_ASTR, KC_LBRC, KC_RBRC, KC_BSLS, XXXXXXX, KC_HOME,
        XXXXXXX, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,             KC_ENT,  KC_END,
        XXXXXXX, KC_LPRN, KC_RPRN, KC_SCLN, KC_DOT,  XXXXXXX, KC_MINS, KC_PLUS, KC_ASTR, KC_SLSH, KC_EQL,           XXXXXXX, XXXXXXX,
        XXXXXXX, XXXXXXX, XXXXXXX,                            XXXXXXX,                   XXXXXXX,          XXXXXXX, XXXXXXX, XXXXXXX
    ),

    /* ALTERNATE Function layer (MOMENTARY)
     * ┌────┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┐
     * │ PWR│F1 │F2 │F3 │F4 │F5 │F6 │   │   │   │   │PRV│NXT│   │VL+│
     * ├────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴───┼───┼───┤
     * │ SLP │F7 │F8 │F9 │F10│F11│F12│   │   │   │   │ PLAY │   │VL-│
     * ├─────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴──┬───┴───┼───┤
     * │ WAKE  │   │   │   │   │   │   │   │   │   │    │ ┌───┐ │   │
     * ├────┬──┴─┬─┴──┬┴───┴───┴───┴───┴───┴───┼───┴┬───┘ │   │ └───┘
     * │RSET│    │    │                        │    │ ┌───┼───┼───┐
     * └────┴────┴────┴────────────────────────┴────┘ │   │   │   │
     *                                                └───┴───┴───┘
     */
    [_LN] = LAYOUT(
        KC_PWR,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_MPRV, KC_MNXT, KC_VOLU,
        KC_SLEP, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,          KC_MPLY, KC_VOLD,
        KC_WAKE, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,          XXXXXXX, XXXXXXX,
        RESET,   XXXXXXX, XXXXXXX,                            XXXXXXX,                   XXXXXXX,          XXXXXXX, XXXXXXX, XXXXXXX
    ),

```